test/support/token_fixture.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Stripe.Fixture.Token do
+
+  def valid_card() do
+    %{card: [
+      number: "4242424242424242",
+      exp_month: exp_month(),
+      exp_year: exp_year(),
+      cvc: "314"
+    ]}
+  end
+
+  def invalid_card() do
+    %{card: [
+      number: "invalid card number",
+      exp_month: exp_month(),
+      exp_year: exp_year(),
+      cvc: "314"
+    ]}
+  end
+
+  defp exp_month() do
```

Comment:
I personally try to avoid magic numbers in tests. So pulling them out a giving it a name is sometimes helpful. It probably doesn't matter in this case though, but I don't think it hurts the code either.